tests/search/embedding/embedding.rb

```diff
@@ -256,6 +282,53 @@ def verify_huggingface_tokens
   end
 
 
+  def check_val_by_idx(expected_v, actual_v, idx)
+      expval = expected_v[idx]
+      actval = actual_v[idx]
+      assert((expval - actval).abs < 1e-5, "#{expval} != #{actval} at index #{idx}")
+      #puts("OK[#{idx}]: #{expval}")
+  end
+
+  def check_prefix_suffix(expected_v, actual_v, fixlen)
```

Comment:
The purpose of this method is just to skip comparison of the middle element (index `5`)?

Reply:
no, it's to skip the middle 758 elements in the actual vector - comparing just the 5 first and last.  index 5 is a placeholder from where the python "torch" library prints "...".